OpenRA.Game/Traits/World/MusicPlaylist.cs

```diff
@@ -45,8 +45,11 @@ public class MusicPlaylist : INotifyActorDisposing, IGameOver
 		readonly MusicInfo[] random;
 		readonly MusicInfo[] playlist;
 
+		public readonly bool IsMusicInstalled;
 		public readonly bool IsMusicAvailable;
 
+		public bool AmbientMusic;
```

Comment:
This must be exposed to Lua. I'm not certain if it should be passed along with the PlayMusic calls or should have an own function to be set.